ratatui-core/Cargo.toml

```diff
@@ -51,6 +55,7 @@ palette = { version = "0.7.6", optional = true }
 paste = "1.0.2"
 serde = { workspace = true, optional = true }
 strum.workspace = true
+thiserror = "2"
```

Comment:
Should we put this behind a feature flag too?

Reply:
Probably not - this one is fairly generally useful (and most dependency trees will have it in them somewhere)